test.js

```diff
@@ -0,0 +1,17 @@
+var expect = require("chai").expect;
+var hello = require('./hello');
+
+describe('hello', function() {
+	it('Should say Hello to name given as a parameter', function() {
+		expect(hello.greet('Brandon')).to.eql("Hello" + " " + "Brandon");
```

Comment:
don't need to separate the strings at the end of this, just use:
`"Hello Brandon"`